src/yb/rpc/messenger.h

```diff
@@ -50,6 +50,8 @@
 #include "yb/rpc/proxy_context.h"
 #include "yb/rpc/scheduler.h"
 
+#include "yb/tserver/pg_client.pb.h"
```

Comment:
not sure if we have dependencies from /rpc to /tserver.

we may create loops if we are not careful.